src/extension/vectortiles/src/main/java/org/geoserver/wms/vector/VectorTileOptions.java

```diff
@@ -0,0 +1,204 @@
+/* (c) 2024 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wms.vector;
+
+import static org.geotools.api.style.FeatureTypeStyle.VT_ATTRIBUTES;
```

Comment:
Did you forget something?
```
Error: 8,981 [ERROR] /home/runner/work/geoserver/geoserver/src/extension/vectortiles/src/main/java/org/geoserver/wms/vector/VectorTileOptions.java:[7,1] cannot find symbol
  symbol:   static VT_ATTRIBUTES
  location: interface org.geotools.api.style.FeatureTypeStyle
```

Reply:
https://github.com/geotools/geotools/pull/4900